projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -122,6 +122,62 @@ class Vectorizer:
     schema: str = "ai"
     table: str = "vectorizer"
 
+    async def run(
```

Comment:
This is merely a copy of the old `cli.run_vectorizer` method but grabbing the data from the vectorizer itself instead of requiring it as arguments as the old function was doing.